tests/cpu/OpOptimizers_tests.cpp

```diff
@@ -490,8 +490,8 @@ OCIO_ADD_TEST(OpOptimizers, optimization2)
     // 3 Range, bit depth conversion identity
     // 4 Matrix, almost identity
     // 5 Range, clamp identity
-    // 6 Lut1D, half domain, raw halfs, identity
-    // 7 Lut1D, raw halfs, identity
+    // 6 Lut1D, half domain, raw halves, identity
```

Comment:
Please revert to `half`

Reply:
Agreed. This refers to the plural of the data type `half`, not the English word "halves" meaning 1/2 of something.